apps/namadillo/src/atoms/chain/atoms.ts

```diff
@@ -83,6 +84,33 @@ export const chainParametersAtom = atomWithQuery<ChainParameters>((get) => {
     queryKey: ["chain-parameters", indexerUrl],
     staleTime: Infinity,
     enabled: !!indexerUrl,
-    queryFn: async () => fetchChainParameters(api),
+    queryFn: async () => {
+      const parameters = await fetchChainParameters(api);
+
+      const unbondingPeriodInEpochs =
```

Comment:
I think this is good! :) But maybe it's worth moving it to some common/utility space(component?) if it's used in multiple places?

Reply:
Thanks! I'll move to a dedicated function

Reply:
Thanks! I'll move it to a dedicated function. But the idea here is to keep it attached to the atom, then anytime we needed, we pull the value directly from the atom and we don't need to care about the math/parameters anymore